app/campaigns/campaigns.db.js

```diff
@@ -0,0 +1,24 @@
+const query = require('./../database')
+
+async function getCampaigns () {
```

Comment:
I think `getAllCampaigns` is a better name to make it more explicit and clear against the other functions